src/utils/track.ts

```diff
@@ -0,0 +1,130 @@
+import { Track, track as _track } from "react-tracking"
+
+// tslint:disable-next-line:no-namespace
+export namespace TrackingInfo {
+  /**
+   * The global tracking-info keys in Artsy’s schema.
+   */
+  export interface Global {
+    /**
+     * The name of an event.
+     */
+    action: string
+
+    /**
+     * The root container component should specify this as the screen context.
+     */
+    page: string
+  }
+
+  export interface Shareable extends Global {
+    /**
+     * The public slug for this entity.
+     */
+    slug: string
+  }
```

Comment:
@mennenia This is an example of what the tracking-info schema could be for items that are ‘shareable’. Please update it to reflect our actual schema.